PaolaSantolloV/react-apprenticeship-capstone1
# Final deliverable

## Description
 This PR contain the final deliverable.

- Change the dark theme

## Type of Change



## How should this be tested?

In your terminal run `npm install` after that `npm run start`, now you can see the page on [http://localhost:3000](http://localhost:3000/)
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## What is the deliverable?

Final deliverable 

## Screenshots and links

Netlify link: [capstoneproject1-final-deliverable.com](https://silly-banach-596f56.netlify.app/)
covergae: 71.33%

![Screen Shot 2022-01-19 at 16 19 29](https://user-images.githubusercontent.com/97254325/150227110-9dc0819b-d622-41a1-bb20-d20e34932c4d.png)

## Notes

If the request are over I could change the API-Key.

## Checklist
- [x ] I added tests to prove that my fix is effective or my deliverable works.
